llvm/docs/vplan-early-exit.dot

```diff
@@ -19,23 +19,27 @@ compound=true
     "middle.split"
   ]
   N4 -> N5 [ label=""]
-  N4 -> N6 [ label=""]
+  N4 -> N7 [ label=""]
```

Comment:
Can't comment on the generated file, it looks like the size in pixels changed, just checking if that was intentional?

Reply:
I didn't specifically try to change the size in pixels - I just ran `dot -Tpng ../llvm/docs/vplan-early-exit.dot`. Is there a recommended command for regenerating this? I couldn't see any documentation about this.

Reply:
Thanks for the suggestion! I've regenerated it with that command and hopefully the image looks bigger now.

Reply:
Oh I didn't realize that there were different defaults on different platforms/version. The larger versions looks good, thanks!